sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -189,7 +189,17 @@
      */
     @Override
     public Flux<ServiceBusReceivedMessage> receiveDeferredMessages(ReceiveMode receiveMode, String sessionId,
-        String associatedLinkName, Iterable<Long> sequenceNumbers) {
+                                                                   String associatedLinkName, Iterable<Long> sequenceNumbers) {
+        if (sequenceNumbers == null) {
+            return fluxError(logger, new NullPointerException("'sequenceNumbers' cannot be null"));
+        }
+
+        final List<Long> numbers = new ArrayList<>();
+        sequenceNumbers.forEach(s -> numbers.add(s));
+
+        if (numbers.isEmpty()) {
+            return Flux.empty();
```

Comment:
We might want to put in `receiveDeferredMessages` API  (which take in `Iterable<Long> sequenceNumbers`  ) in sync and async client that empty list will complete the flux without sending to service bus .

Reply:
Thinking about this a bit more - are we sure that's something that needs documenting? We're no longer treating it as an error condition so this just seems to fall under normal operating conditions.